x-pack/plugins/lens/public/datatable_visualization/components/table_basic.scss

```diff
@@ -11,3 +16,7 @@
 .lnsDataTable__filter:focus-within {
   opacity: 1;
 }
+
+.lnsDataTableCellContent {
+  @include euiTextTruncate;
+}
```

Comment:
This seems unnecessary, as `EuiDataGrid` already truncates cell contents. Am I correct in assuming this style was added because the `EuiDataGrid` truncation stopped working when wrapping the cell contents with `div.lnsDataTableCellContent`? If that assumption is correct, we can fix this in one of two ways:

- If this `div` isn't really necessary, simply removing it will allow the `EuiDataGrid` truncation to work properly again. 
- Alternatively, if wrapping element is necessary, changing it to an inline element (such as a `span`) should also allow the `EuiDataGrid` truncation to work properly.